java/src/jmri/jmrit/operations/trains/tools/PrintShowCarsInTrain.java

```diff
@@ -31,12 +33,17 @@ public class PrintShowCarsInTrain {
             InstanceManager.getDefault(CarRoads.class).getMaxNameLength() + Control.max_len_string_road_number;
     static final String TAB = padString("", fieldSize);
 
-    public PrintShowCarsInTrain(Train train, boolean isPreview) {
+    public PrintShowCarsInTrain() {
         super();
-        printCarsInTrain(train, isPreview);
     }
```

Comment:
If this is the only constructor in the class and it only calls super(), I don't think you need it at all.